src/content/community/support/index.md

```diff
@@ -0,0 +1,77 @@
+description: Get support in the Ethereum ecosystem.
+sidebar: true
+lang: en
+---
+
+# Ethereum support {#support}
+
+## Official Ethereum support {#official-support}
+
+Are you looking for the official Ethereum support? The first thing you should know is that Ethereum is decentralized. This means no central organization, entity, or person owns Ethereum, and because of this, no official support channels exist.
+
+Understanding the decentralized nature of Ethereum is vital because anyone claiming to be official support for Ethereum is probably trying to scam you! The best protection against scammers is educating yourself and taking security seriously.
+
+<DocLink to="/security/" title="Ethereum security and scam prevention" />
+
+<DocLink to="/learn/" title="Learn Ethereum fundamentals" />
+
+Despite the lack of official support, many groups, communities, and projects across the Ethereum ecosystem are happy to help.
+
+## Wallet support {#wallet-support}
+
+Having trouble with your wallet? Most wallets have dedicated support teams that can help.
+
+- [MetaMask](https://metamask.zendesk.com/hc/)
+- [Argent](https://support.argent.xyz/hc/)
+
+_This is not an exhaustive list. Need help finding support? Join the [ethereum.org discord](https://discord.gg/rZz26QWfCg) and we'll try to help._
+
+## Building dapps {#building-support}
+
+Building can be hard. Here are some development focused spaces with experienced Ethereum developers that are happy to help.
+
+- [CryptoDevs discord](https://discord.gg/Z9TA39m8Yu)
+- [Ethereum StackExchange](https://ethereum.stackexchange.com/)
+- [StackOverflow](https://stackoverflow.com/questions/tagged/web3)
+
+### Tooling {#dapp-tooling}
+
+Does your question relate to a particular tool, project, or library? Most projects have chat servers or forums dedicated to supporting you.
+
+- [Solidity](https://gitter.im/ethereum/solidity/) <!-- What is the best place to send people to here? -->
+- [ethers.js](https://discord.gg/6jyGVDK6Jx)
+- [web3.js](https://discord.gg/GsABYQu4sC)
+- [Hardhat](https://discord.gg/xtrMGhmbfZ)
+- [Truffle](https://discord.gg/8uKcsccEYE)
+- [Chainlink](https://discord.gg/2YHSAey)
+- [Optimism](https://discord.gg/wGt8hjbZtu)
+
+## Running a node {#node-support}
+
+If you're running a node or validator, here are some communities that are dedicated to helping you get started.
+
+- [EthStaker discord](https://discord.io/ethstaker)
+- [EthStaker reddit](https://www.reddit.com/r/ethstaker)
+
+Most of the teams building Ethereum clients also have dedicated, public-facing, spaces where you can get support and ask questions.
+
+### Execution clients {#execution-clients}
+
+- [Geth](https://discord.gg/FqDzupGyYf)
+- [Nethermind](https://discord.gg/YJx3pm8z5C)
+- [Besu](https://discord.gg/p8djYngzKN)
+- [Ergion](https://github.com/ledgerwatch/erigon/issues) <!-- Does Erigon have a public facing channel? -->
+
+### Consensus clients {#consensus-clients}
+
+- [Prysm](https://discord.gg/prysmaticlabs)
```

Comment:
In the spirit of diversity, is there a way to randomize this? Probably not in markdown, just curious.

Reply:
Mmm not sure how we would do that. But if possible to keep the content in markdown but randomize somehow I say lets do it :-)

Reply:
Would consider listing Prysm lower solely for the sake of subtly promoting client diversity. We should try to emphasize the other client communities.